corehq/pillows/mappings/case_mapping.py

```diff
@@ -206,13 +194,9 @@
                 "exact": {
                     "index": "not_analyzed",
                     "type": "string"
-                },
-                "type": {
```

Comment:
The "deleted" fields [are the "top level" property](https://www.elastic.co/guide/en/elasticsearch/reference/1.3/_multi_fields.html). I say "deleted" because they're not really deleted.  The old `multi_field` type had a [special behavior](https://www.elastic.co/guide/en/elasticsearch/reference/0.90/mapping-multi-field-type.html#_accessing_fields) that the field _named the same as the property_ (the "default" field) was the one accessible without its `.<fieldname>` suffix.  Elasticsearch versions 1.x and 2.x have always silently upgraded mappings to the current format (for example, see: http://localhost:9200/smslogs/_mapping/field/domain), but version 5.x and up strictly prohibit the `multi_field` type.

Reply:
Thanks! That makes a lot of sense